docs/manage-search-index.md

```diff
@@ -0,0 +1,50 @@
+# Manage Search Index
+
+Each document is stored in the index, and is the type and ID. The document is a JSON object that contains zero or more fields or key-value pairs. The original JSON document indexing will be stored in the field_source that returns a default receipt or document search.
+
+The analysis is performed by а custom analyzer defined by index. Token filter:
+- custom_edge_ngram (default) 
+- custome_ngram
+
+Mapping is used to define how a document, and the fields it contains, are stored and indexed.
```

Comment:
Maybe "Mapping is used to define how a document and its fields are stored and indexed."?

Reply:
OK!